docs/src/languages/java/_index.md

```diff
@@ -116,6 +118,10 @@ highlight=java
 
 {{< /codetabs >}}
 
+ {{< note >}}
+Those samples are using the [Configuration Reader](https://github.com/platformsh/config-reader-java) that Platform.sh provides to you if you wish, however, if your favorite Java frameworks allow you to overwrite the configurations following [The Twelve-Factor App](https://12factor.net/config) such as Spring, Micronaut, Quarkus, Jakarta EE/MicroProfile, and so on. 
```

Comment:
```suggestion

Those samples are using the [Configuration Reader](https://github.com/platformsh/config-reader-java) that Platform.sh provides to you if you wish, however, if your favorite Java frameworks allow you to overwrite the configurations following [The Twelve-Factor App](https://12factor.net/config) such as Spring, Micronaut, Quarkus, Jakarta EE/MicroProfile, and so on. 

```

Reply:
The  sentence  seems like its missing an ending: "(use provided config  reader), however (if  frameworks allow overwrite along 12FA like A,B, C),  then ...

Reply:
Thank you, I'll accept your suggestion and then refactor it.